frontend/src/models/associations/election.ts

```diff
@@ -2,23 +2,38 @@ import { User } from "../user";
 import { Association } from "./association";
 
 export interface Choice {
-  id: string;
-  election: Election;
+  id: number;
+  name: string;
+  numberOfOfflineVotes: number;
+  numberOfOnlineVotes: number;
```

Comment:
The camel case of `offline` and `online` is not consistent with frontend/src/components/associations/elections/view/View.tsx